### DISCUSSION TITLE
How do I fix this Build error in Unity? I have no problem running the game inside Unity but I have a problem trying to export it as .exe

### DISCUSSION THREAD

**Discussion by annwon603** [community/community#77751] [Programming Help] [answered]
### Body

![image](https://github.com/community/community/assets/101071975/f1842bd7-0ba1-4a53-9303-f2a01c745ba0)
![image](https://github.com/community/community/assets/101071975/670f353a-c747-49b3-ad26-4d3da2bf6e1c)
I got these errors so far after fixing my C# scripts. I'm trying to export my game as a Windows game. 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by DylanDevelops** [comment]
I notice that the first screenshot says that you have compile errors. Did you go through your scripts and fix any of those? If you click play in the editor, are you able to run your game?

**Reply by annwon603** [reply]
Yes, I did fix my scripts. I was also able to run the game when I clicked play in my editor

**Reply by DylanDevelops** [reply]
Awesome. If my answer was correct you might wanna mark it as the answer so others can find a solution if they have the same problem.

**Reply by annwon603** [reply]
Oh, I forgot to mention that my game can run in my editor, it still won't let me build it.

**Reply by DylanDevelops** [reply, marked as answer]
It looks like the errors are saying that a file could not be found. Double-check files exist for any plugins you might use.

**Reply by annwon603** [reply]
oh ok, I can now build it after updating the burst from the package manager.

**Reply by DylanDevelops** [reply]
Glad to hear it! Good luck with your game!
